README.md

```diff
@@ -295,14 +301,14 @@ __Arguments__
  Gives Redbird a callback function with two parameters, the ExpressJS request 
```

Comment:
Its not a expressJs object but just the standard http request and response objects: https://nodejs.org/api/http.html#http_class_http_clientrequest https://nodejs.org/api/http.html#http_class_http_serverresponse